check_modulemd.py

```diff
@@ -66,23 +66,52 @@ def setUp(self):
         self.mdfile = mdfile
         self.mmd = mmd
 
-        self.chkr = None
+    def _init_spell_checker(self):
+        """
+        Initialize spell checker dictionary
+        """
+
+        default_dict = "en_US"
+        spell_dict = None
+
         jargonfile = self.params.get('jargonfile')
-        dict = ""
         if not jargonfile:
             jargonfile = os.environ.get('JARGONFILE')
-        try:
-            if jargonfile is not None:
+        if jargonfile is not None:
+            try:
                 jargonfile = str(jargonfile)
-                dict = DictWithPWL("en_US", jargonfile)
-                for w in self.mmd.name.split('-'):
-                    dict.add_to_session(w)
-                self.chkr = SpellChecker(dict)
-            else:
-                self.chkr = SpellChecker("en_US")
+                spell_dict = DictWithPWL(default_dict, jargonfile)
+            except:
+                self.error(
+                    "Could not initialize dictionary using %s file" % jargonfile)
+
+        if not spell_dict:
+            try:
+                spell_dict = DictWithPWL(default_dict)
```

Comment:
Apologies. No need to update. DictWithPWL() is new to me and I misread the spec. But every example I could find of its usage always included two or more arguments--with the second arg possibly being None.